tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -19,6 +19,11 @@ parameters:
   type: string
   default: ''
 
+- name: QnnSDKVersion
+  displayName: QNN SDK Version
+  type: string
+  default: '2.27.0.240926'
```

Comment:
FYI about https://github.com/microsoft/onnxruntime/pull/22724 which is changing the version.

Reply:
we are updating to 2.28.0.241029